doc/api/https.md

```diff
@@ -240,11 +240,6 @@ Global instance of [`https.Agent`][] for all HTTPS client requests.
 <!-- YAML
 added: v0.3.6
 changes:
-  - version:
-    - v14.1.0
-    - v13.14.0
-    pr-url: https://github.com/nodejs/node/pull/32786
-    description: The `highWaterMark` option is accepted now.
```

Comment:
We should likely keep this note and add a new changes entry that mentions that it's removed again.

The same applies to the other changes entry.

Reply:
Good point. Done.